### ISSUE TITLE
VictoriaLogs: How to save a search?

### ISSUE THREAD

**Issue by leandrok** [VictoriaMetrics/VictoriaMetrics#7120]
### Is your question request related to a specific component?

VictoriaLogs UI

### Describe the question in detail

I have deployed VictoriaLogs. I'm using the UI. I can type searches in the "Log query" textbox and execute them. So far so good.
Now, can I save a search for future usage? I don't see that option. Thanks

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/troubleshooting/
- [ ] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by tiny-pangolin** [comment]
I do not believe there is a way to save a query directly in Victorialogs. There are two workarounds you can use to access frequently used queries.

1. bookmark the urls and share links for frequency used queries
2. Use the [VictoriaLogs Grafana data source](https://github.com/VictoriaMetrics/victorialogs-datasource?tab=readme-ovfile#installation) and make a Grafana Dashboard

I will convert this to a feature request and see what the team thinks

**Comment by leandrok** [comment]
Thanks for your info and thanks for converting this into a feature request. I'm sure this will be very useful, thanks

**Comment by arturminchukov** [comment]
Close as duplicate. See: #8500

**Comment by arturminchukov** [comment]
Close as duplicate. See #8500
